packages/protocol/docs/Derivation.md

```diff
@@ -177,37 +187,64 @@ struct BlobSlice {
 }
 ```
 
-The `BlobSlice` struct represents binary data distributed across multiple blobs. Processing involves the following steps:
+The `BlobSlice` struct represents binary data distributed across multiple blobs. Each `DerivationSource` in the `derivation.sources[]` array is processed independently to extract its `DerivationSourceManifest`.
+
+**TODO**: The proposal-level `proverAuthBytes` field needs a storage location. Current `Derivation` struct only has per-source `blobSlice` fields. Option: add dedicated `proposalBlob` field to `Derivation` struct (requires protocol change).?
```

Comment:
If we want to avoid protocol changes, maybe we can let proposer submit `ProposalManifest` in `BlobSlice`, and force-inclusion submitter submit `DerivationSourceManifest`?